numpy/core/code_generators/numpy_api.py

```diff
@@ -5,7 +5,8 @@
 
 Whenever you change one index, you break the ABI (and the ABI version number
 should be incremented). Whenever you add an item to one of the dict, the API
-needs to be updated.
+needs to be updated in both setup_common.py and by adding an appropriate
+entry to cversion.txt (generate the hash via "python cversions.py".
```

Comment:
nit: missing paren

Reply:
What is "best practice" when fixing a merged PR, a new PR or modifying this one again?

Reply:
You can't modify an existing PR once it's merged.

What I normally do is keep working in the same branch, but open a new PR

Reply:
fixed (in doc-nditer PR)